lib/ansible/modules/packaging/language/maven_artifact.py

```diff
@@ -163,7 +163,7 @@ def __init__(self, group_id, artifact_id, version, classifier=None, extension='j
         self.group_id = group_id
         self.artifact_id = artifact_id
         self.version = version
-        self.classifier = classifier
+        self.classifier = classifier if classifier else ''
```

Comment:
Why this guard condition and not as default in the `__init__` signature right above?